go-controller/pkg/cni/cni.go

```diff
@@ -195,6 +197,10 @@ func (pr *PodRequest) cmdDel(clientset *ClientSet) (*Response, error) {
 
 			// Delete the DPU connection-details annotation
 			_ = pr.updatePodDPUConnDetailsWithRetry(&kube.Kube{KClient: clientset.kclient}, clientset.podLister, nil)
+
+			if pr.IsVFIO {
+				return response, nil
+			}
```

Comment:
I don't think this is needed.

Just let the code go through here, since you handle things in UnconfigureInterface()